src/functions/boundaryAreaOverlap.ts

```diff
@@ -66,15 +66,17 @@ export async function boundaryAreaOverlap(
     return sketchToMpaClass[sketchMetric.sketchId!];
   };
 
-  const levelMetrics = await overlapAreaGroupMetrics({
-    metricId: metricGroup.metricId,
-    groupIds: ["FULLY_PROTECTED", "HIGHLY_PROTECTED"],
-    sketch: clippedSketch,
-    metricToGroup: metricToLevel,
-    metrics: areaMetrics,
-    classId: metricGroup.classes[0].classId,
-    outerArea: totalAreaMetric.value,
-  });
+  const levelMetrics = (
+    await overlapAreaGroupMetrics({
+      metricId: metricGroup.metricId,
+      groupIds: ["FULLY_PROTECTED", "HIGHLY_PROTECTED"],
+      sketch: clippedSketch,
+      metricToGroup: metricToLevel,
+      metrics: areaMetrics,
+      classId: metricGroup.classes[0].classId,
+      outerArea: totalAreaMetric.value,
+    })
+  ).map((metric) => ({ ...metric, geographyId }));
```

Comment:
`geographyId` is now being appended to all metrics in the gp functions, much like classId is.